app/models/communication.rb

```diff
@@ -3,6 +3,7 @@ class Communication < ApplicationRecord
   include Hashable
 
   belongs_to :appointment
+  belongs_to :appointment_reminder, optional: true
```

Comment:
One more time =) ...if we migrate the existing system to use appointment_reminder, this can at least be enforced as required going forward, and I think we could actually remove the direct relationship to appointment (as appointment reminder gives us that instead.